### PR TITLE
feat: Create sig-marketing

### PR DESCRIPTION
**Pull request by todaywasawesome**
Sig marketing has been operating less formally since before ArgoCon '21. It's time to formalize!

Signed-off-by: Dan Garfield <dan@codefresh.io>

### CODE REVIEW

**Comment by christianh814** [comment]
+1 🙌

File: sigs/sig-marketing/README.md
Comment (wanghong230) [inline sigs/sig-marketing/README.md:1]: This is great progress. Here are some additional ideas.
- It is critical to call out the responsibilities, boundaries, rules for the marketing effort here.
- The marketing group should be to promote the Argo Open Source brand.
- I understand all the marketing professionals are from vendors, as the contributors are mainly developers. So, everyone including us will face the balance and conflict every now and then between the vendor initiative and open source. So I recommend limiting the vendor max seats to 2 as membership to have a balanced conversation.
- As more vendors are behind the curtain now and building the association with the argo product. I am pretty sure such interests could come and go based on the vendor's true interest level. It is crucial to call out the Sig Marketing responsibilities to ensure the longevity of the Argo brand, rather than a springboard.

File: sigs/sig-marketing/README.md
Comment (wanghong230) [inline sigs/sig-marketing/README.md:7]:
```suggestion
## Membership Requirements 
- Must represent the open-source community’s best interest with a vendor-neutral practice.
- Must follow the CNCF rules and guidances closely.
- Must have shown a proven record of enabling the open-source community by marketing.
- Technical marketing is preferred.
- Cannot have more than two members from the same company.
- The membership will be revisited every calendar year.
```

**Comment by todaywasawesome** [comment]
Per @edlee2121's request, we should create a general practices document for sigs.

File: sigs/sig-marketing/README.md
Comment (hblixt) [reply sigs/sig-marketing/README.md:7]: I believe that membership for all our SIGs should be open to anyone and everyone who wants to engage with a particular SIG. The SIGs are really working groups with a given charter and IMO, there's really no need to institute election/voting/membership procedures for them. All decisions should be consensus driven, and if consensus cant be reached, the maintainer group has the final say, as per our governance.

File: sigs/sig-marketing/social-media-guidelines.md
Comment (hblixt) [inline sigs/sig-marketing/social-media-guidelines.md:38]: We should encourage post from everyone, no matter if they work for a vendor or not, but blogs/tweets need to be neutral, e.g. a promoted blog or video can't have one section about Argo and another section about a product offering.

File: sigs/sig-marketing/social-media-guidelines.md
Comment (hblixt) [reply sigs/sig-marketing/social-media-guidelines.md:20]: I think a workshop/webinar from a vendor is fine, as long as that workshop is focused on the community version, e.g. you can't have a 60 min workshop that talks 50 mins about Argo and then 10 mins about how Argo works in/on your platform. It has to be 100% vendor neutral. 
I'd take out certifications, unless we are planning to do CNCF certifications. Vendor certification marketing should not be allowed.

**Review by wojtekidd — CHANGES_REQUESTED** [review]

File: sigs/sig-marketing/social-media-guidelines.md
Comment (edlee2121) [inline sigs/sig-marketing/social-media-guidelines.md:12]:
```suggestion
* All social media posts by members related to the Argo Project whether from official, company or personal accounts must follow the below guidelines.
```

**Review by wojtekidd — COMMENTED** [review]
I understand we will be able to include suggested changes and finish it only when the Argo SIG document by Ed Lee will be finished an accepted?

**Comment by todaywasawesome** [comment]
> I understand we will be able to include suggested changes and finish it only when the Argo SIG document by Ed Lee will be finished an accepted?

I don't think it needs to be completed. As long as the charter doesn't express powers beyond what is expected.

File: sigs/sig-marketing/README.md
Comment (hblixt) [inline sigs/sig-marketing/README.md:14]: I dont think we need to limit it to two / company. Matters should be consensus driven and for anything that cant reach consensus, it should be brought to and follow maintainer voting procedures. 
Membership here is separate than being a maintainer, so adding to SIG Marketing wont add to the maintainer group.

File: sigs/sig-marketing/social-media-guidelines.md
Comment (hblixt) [inline sigs/sig-marketing/social-media-guidelines.md:13]: I think it would be good to have some language about outside input before posting, e.g. 

"Posts and content should be approved by a member from a **_different_** organization before being published. "

**Comment by edlee2121** [comment]
Here's the general SIG-GOVERNANCE doc.
https://github.com/argoproj/argoproj/pull/144

**Review by edlee2121 — APPROVED** [review]

**Comment by edlee2121** [comment]
The general SIG-GOVERNANCE doc has been merged to main.
https://github.com/argoproj/argoproj/pull/144

File: sigs/sig-marketing/social-media-guidelines.md
Comment (hblixt) [inline sigs/sig-marketing/social-media-guidelines.md:26]: The goal of the project social accounts is to promote the project and related content. Any content posted, or linked to, should be relevant, no more than a week old and not be re-posts. 
Linking to vendor blogs and/or content should be restricted to once / month / vendor.

File: sigs/sig-marketing/social-media-guidelines.md
Comment (hblixt) [inline sigs/sig-marketing/social-media-guidelines.md:38]: * Cross project blogs or content together with other open source projects that integrate with any of the Argo projects.

File: sigs/sig-marketing/README.md
Comment (hblixt) [inline sigs/sig-marketing/README.md:17]: * Secretary/Coordinator - Coordinates work in the SIG. Leads and runs the SIG Marketing meetings. Reports back on SIG Marketing completed and upcoming work to maintainers.

File: sigs/sig-marketing/README.md
Comment (hblixt) [inline sigs/sig-marketing/README.md:28]: - Remove Daniel

**Review by hblixt — CHANGES_REQUESTED** [review]

**Review by wojtekidd — APPROVED** [review]

**Comment by todaywasawesome** [comment]
I would suggest removing the vendor language entirely. The official social
media policy should be promoting good Argo content.

File: sigs/sig-marketing/social-media-guidelines.md
Comment (wojtekidd) [inline sigs/sig-marketing/social-media-guidelines.md:43]:
```suggestion
    * Cross-project blogs or content together with other open source projects that integrate with any of the Argo projects
```

**Review by wojtekidd — CHANGES_REQUESTED** [review]

**Comment by edlee2121** [comment]
Can we merge/close this?

**Comment by todaywasawesome** [comment]
> Can we merge/close this?

Yes please, the committee is functioning and has been for quite some time. Let's make it official!

**Review by wojtekidd — APPROVED** [review]

**Comment by wojtekidd** [comment]
@hblixt I think you have to add your review to make this work 😉

**Comment by christianh814** [comment]
There are still pending suggestions for this PR

**Comment by hblixt** [comment]
> There are still pending suggestions for this PR

I think everything is resolved now?

**Review by hblixt — APPROVED** [review]
I believe everything has been addressed and since this has been functioning for a while now, time to make it official :)

**Comment by christianh814** [comment]
> > There are still pending suggestions for this PR
> 
> I think everything is resolved now?

All that needs to change is my email

**Comment by christianh814** [comment]
> > > > There are still pending suggestions for this PR
> > > 
> > > 
> > > I think everything is resolved now?
> > 
> > 
> > All that needs to change is my email
> 
> Can we do it post merge? Because it's ready to merge and I don't want to trigger review from everyone.

Sure, I can PR after the merge.

**Pull request merged**
